src/audio/mfcc/mfcc_psy.c

```diff
@@ -0,0 +1,281 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
These Mel related functions should also fit new library for generic psychoacoustics functions.